src/app/screens/Login/index.js

```diff
@@ -24,26 +26,24 @@ export default function Login({ history }) {
   );
 
   const handleSignUp = useCallback(() => {
-    // eslint-disable-next-line no-console
-    console.log('se logueo con exito', response);
-  }, [response]);
+    history.push(ROUTES.signUp);
+  }, [history]);
 
   useEffect(() => {
     if (response) {
       if (isError) {
         addToast(response, { appearance: 'error' });
       } else {
-        handleSignUp();
+        saveStorage(response.accessToken, 'accessToken');
+        history.go();
```

Comment:
This is not necessary, with the push its ok.

Also you have the internal state management missing. One thing is to have the token stored in LS, and another one is having it available to the app and make it reactive to changes, when the variable is null, it will automatically redirect you when you push routes to the login